### PR TITLE
Add units to some things

### PR DESCRIPTION
**Pull request by chazlarson**
## Description

Adds units to a couple entries in the wiki.

## Type of Change

- [X] Documentation change (non-code changes affecting only the wiki)

## Checklist

- [X] My code was submitted to the nightly branch of the repository.

### CODE REVIEW

**Comment by chazlarson** [comment]
the spellcheck failure here is addressed in #2038

File: docs/kometa/environmental.md
Comment (YozoraXCII) [inline docs/kometa/environmental.md:1]: misspelled word: expfessed

File: docs/config/plex.md
Comment (YozoraXCII) [inline docs/config/plex.md:1]: Plex' should say Plex's I think

**Review by YozoraXCII — COMMENTED** [review]
Added comments

**Comment by chazlarson** [comment]
backing this out for the moment

**Pull request closed without merging**